service/src/main/resources/HTMLTemplate.html

```diff
@@ -1840,6 +1848,25 @@ <h3 class="heading heading--two">Other {{ counter }}</h3>
 
   <h3 class="heading heading--two">Grounds for the application</h3>
   <dl class="data-list">
+    {% if case_data.orders is not empty and
+    case_data.orders.orderType is not empty and
+    case_data.orders.orderType contains 'EMERGENCY_PROTECTION_ORDER' %}
+      <div class="data-list__row">
+        <dt class="data-list__title">Grounds for emergency protection order</dt>
+        <dd class="data-list__definition">
+          {% if case_data.groundsForEPO is not empty and
+          case_data.groundsForEPO.EPO_reason is not empty %}
```

Comment:
We should rename `EPO_reason` to something simpler. A `EPO` prefix is not needed because we operate within context of `GroundsForEPO` object. Maybe simple `reason` or `grounds` be enough.